src/hooks/UseThrottleFunction/API/API.tsx

```diff
@@ -1,6 +1,7 @@
-import React, { useState } from 'react'
+import type React from 'react'
+import { useState } from 'react'
 import useThrottleFunction from '../useThrottleFunction'
-import COUNTRIES from './assets/countries.json'
+import type COUNTRIES from './assets/countries.json'
```

Comment:
It is my opinion that this is not a type

Reply:
It must've been added because we do `keyof typeof COUNTRIES` below. Fixed and pushed.